MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -809,7 +830,29 @@
 		6441B1691EFC64E50076499F /* WaypointConfirmationViewController.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; name = WaypointConfirmationViewController.swift; path = Example/WaypointConfirmationViewController.swift; sourceTree = "<group>"; };
 		64847A031F04629D003F3A69 /* Feedback.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = Feedback.swift; sourceTree = "<group>"; };
 		7C12F2D7225B7C310010A931 /* DCA-Arboretum-dummy-faster-route.json */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.json; path = "DCA-Arboretum-dummy-faster-route.json"; sourceTree = "<group>"; };
+		8A07D26325C9E517005E1790 /* StyledFeature.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = StyledFeature.swift; sourceTree = "<group>"; };
```

Comment:
StyledFeature.swift should be moved to the Example/ folder.

Reply:
Good catch. Fixed.